pkg/generated/openapi/def.bzl

```diff
@@ -1,5 +1,6 @@
 load("@io_bazel_rules_go//go:def.bzl", "go_library")
 load("@io_kubernetes_build//defs:go.bzl", "go_genrule")
+load("//build:openapi.bzl", "openapi_target_prefix", "openapi_output_prefix", "openapi_pkg_prefix")
```

Comment:
I think this will always load the one in this repository: //build:openapi.bzl is a target in the current repository.

I would think these would have to be parameters to the rule.

Reply:
That was the intent - it would allow a vendoring repo to define //build/openapi.bzl to configure the generation of vendored kube's openapi file.  As per our discussion online, I've switched to loading //build/openapi.bzl in the BUILD file and updated openapi_library to accept the paramters.